deploifai/api/__init__.py

```diff
@@ -410,3 +409,213 @@ def create_project(self, project_name: str, cloud_profile: CloudProfile, fragmen
+        mutation($whereProject: ProjectWhereUniqueInput! $data: CreateTrainingInput!){
+            createTraining(whereProject: $whereProject data: $data){
+            name
+            status
+            }
+        }
+        """
+
+        if falcon_id:
+            if data_storage_id:
+                variables = {
+                    "whereProject": {"id": project_id},
+                    "data": {
+                        "name": name,
+                        "cloudProfileId": cloud_profile_id,
+                        "cloudProviderFalconConfig": {
+                            "plan": falcon_plan,
+                            "usesGpu": falcon_gpu,
+                        },
+                        "falconMLConfigId": falcon_id,
+                        "dataStorageIds": data_storage_id,
+                    },
+                }
+            else:
+                variables = {
+                    "whereProject": {"id": project_id},
+                    "data": {
+                        "name": name,
+                        "cloudProfileId": cloud_profile_id,
+                        "cloudProviderFalconConfig": {
+                            "plan": falcon_plan,
+                            "usesGpu": falcon_gpu,
+                        },
+                        "falconMLConfigId": falcon_id,
+                    },
+                }
+        else:
+            if data_storage_id:
+                variables = {
+                    "whereProject": {"id": project_id},
+                    "data": {
+                        "name": name,
+                        "cloudProfileId": cloud_profile_id,
+                        "cloudProviderFalconConfig": {
+                            "plan": falcon_plan,
+                            "usesGpu": falcon_gpu,
+                        },
+                        "dataStorageIds": data_storage_id,
+                    },
+                }
+            else:
+                variables = {
+                    "whereProject": {"id": project_id},
+                    "data": {
+                        "name": name,
+                        "cloudProfileId": cloud_profile_id,
+                        "cloudProviderFalconConfig": {
+                            "plan": falcon_plan,
+                            "usesGpu": falcon_gpu,
+                        },
+                    },
+                }
+        try:
+            r = requests.post(
+                self.uri,
+                json={"query": mutation, "variables": variables},
+                headers=self.headers,
+            )
+
+            create_mutation_data = r.json()
+
+            return create_mutation_data["data"]["createTraining"]
+        except TypeError:
+            raise DeploifaiAPIError("Could not create Training Server. Please try again.")
+        except KeyError:
+            raise DeploifaiAPIError("Could not create Training Server. Please try again.")
+
+    def get_server(self, workspace: str):
```

Comment:
again, consistency is key when it comes to variable and function naming. This should be `get_training_servers`